src/firmware/app/primitives/move_new_primitive.c

```diff
@@ -0,0 +1,46 @@
+#include "firmware/app/primitives/move_new_primitive.h"
+
+#include <math.h>
+#include <stdio.h>
+#include <stdlib.h>
+
+#include "firmware/app/primitives/move_manager.h"
+
+void app_move_primitive_start(MovePrimitiveMsg prim_msg, void *void_state_ptr,
```

Comment:
Are we not deleting the old move primitive as part of this PR? Any reason to temporarily keep both around?

Reply:
until https://github.com/UBC-Thunderbots/Software/issues/1643 is done, we're still using the old primitiveparams, so to avoid name conflicts, I'm making a new moveprimitive that uses the new primitivemsg that can be swapped in later

Reply:
Why not just move over one primitive at a time? :thinking:

Reply:
yeah I can delete move_new_primitive